msal4j-sdk/src/test/java/com/microsoft/aad/msal4j/OnBehalfOfTests.java

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+import static org.mockito.Mockito.times;
+
+@ExtendWith(MockitoExtension.class)
+class OnBehalfOfTests {
+
+    private String getSuccessfulResponse(String accessToken) {
+        return "{\"access_token\":\""+accessToken+"\",\"expires_in\": \""+ 60*60*1000 +"\",\"token_type\":" +
+                "\"Bearer\",\"client_id\":\"client_id\",\"Content-Type\":\"text/html; charset=utf-8\"}";
```

Comment:
1. Consider adding this to a common file. It's the standard /token response for S2S
2. OBO receives id tokens, refresh tokens etc. The response should reflect that. 
3. I am not aware of the COntent-Type being part of the response content. It may be a header?

Reply:
This was just copied from some similar tests in ManagedIdentityTests, and what I was looking at in these new tests didn't need anything different in the response so I didn't look at it too closely.

In the latest commit I moved it to TestHelper and restructured it to give it a lot more flexibility: the string is now properly formatted with placeholders, and the placeholders are optionally filled in based on values set in a map.